blog.css

```diff
@@ -0,0 +1,316 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --font-secondary: "Bakbak One", Times New Roman, Georgia, serif;
+}
+
+img {
+  max-width: 100%;
+  height: auto;
+  vertical-align: top;
+}
+body {
+  background-color: #0d0d15;
+  margin: 0;
+  font-family: "Poppins", Arial, sans-serif;
+  color: #f5f8fb;
+  font-size: 18px;
+  line-height: 28px;
+}
+
+header {
+  padding: 32px;
+  text-align: start;
+  display: flex;
+}
+
+.logo {
+  left: 52px;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.navigation-list {
+  list-style-type: none;
+  display: flex;
+}
+
+.nav-link {
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 24px;
+  text-decoration-line: none;
+  color: #f5f8fb;
+}
+
+.btn {
+  color: #f5f8fb;
+  font-size: 18px;
+  line-height: 28px;
+  font-weight: 600;
+  text-decoration-line: none;
+  text-align: center;
+  border-width: 2px;
+  border-style: solid;
+  border-color: rgba(208, 215, 251, 0.17);
+  height: 48px;
+  padding: 10px 24px;
+}
+
+.button-connect {
+  background-color: #0265dc;
+  box-sizing: border-box;
+  position: relative;
+}
+
+h1 {
+  font-family: var(--font-secondary);
+  font-size: 52px;
+  font-weight: 400;
+  line-height: 62px;
+  letter-spacing: -0.5px;
+  text-align: left;
+  color: #f5f8fb;
+}
+.blog-nav {
+  list-style-type: none;
+}
+.blog-nav a {
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 20px;
+  text-decoration: none;
+  color: #f5f8fb;
+}
+.blog-nav > li:last-child a {
+  color: #e7eef7a8;
+}
+.sidebar-box {
+  width: 328px;
+  height: 384px;
+  background-color: #1d1c28;
+}
+.sidebar-box-tag {
+  width: 328px;
+  height: 184px;
+  background-color: #1d1c28;
+}
+
+.sidebar-title {
+  font-family: var(--font-secondary);
+  font-size: 24px;
+  font-weight: 400;
+  line-height: 32px;
+  color: #f5f8fb;
+  border-bottom: 1px solid #bfc5ee1f;
+}
+
+.sidebar-list {
+  list-style-type: none;
+}
+
+.sidebar-link {
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 20px;
+  text-decoration: none;
+  color: #f5f8fb;
+}
+
+.sidebar-list-tag {
+  list-style-type: none;
+  display: flex;
+  gap: 8px;
+}
+
+.sidebar-link-tag {
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 20px;
+  text-decoration: none;
+  color: #f5f8fb;
+}
+
+.sidebar-link-tag {
+  border: 2px solid #d0d7fb2b;
+  padding: 6px 12px 6px 12px;
+}
+
+.blog-post {
+  width: 920px;
+  height: 2, 528px;
```

Comment:
Please check the value, it should be amount with 'px' or '%'

Reply:
done